code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -106,15 +106,20 @@
 	item_state = "laceups"
 
 /obj/item/clothing/shoes/galoshes
-	desc = "Rubber boots"
 	name = "galoshes"
+	desc = "Rubber boots"
 	icon_state = "galoshes"
 	permeability_coefficient = 0.05
 	flags = NOSLIP
 	slowdown = SHOES_SLOWDOWN+1
 	species_fit = list(VOX_SHAPED)
 	heat_conductivity = INS_SHOE_HEAT_CONDUCTIVITY
 
+/obj/item/clothing/shoes/galoshes/vault/broken
```

Comment:
So people know what the fuck they're used for.  I'm torn honestly.  I want to put it in the vault file but i hate jamming stuff into random files and making it hard to find.  I can drop the vault path if you think its unneeded.